hdkeychain/extendedkey_test.go

```diff
@@ -893,3 +893,240 @@ func TestZero(t *testing.T) {
 		}
 	}
 }
+
+// TestBIP32Std ensures that deriving children from keys with a leading zero
+// byte with ChildBIP32Std is done according to the BIP-32 standard, and that
+// the Child method generates a backwards-compatible result. This uses the same
+// test seed and path as the btcutil test, TestLeadingZero, to ensure consistent
+// results.
+func TestBIP32Std(t *testing.T) {
```

Comment:
I guess it doesn't hurt to add this, but I'm not really sure it's necessary.  The expected Decred results with a 0 high byte are already tested [here](https://github.com/decred/dcrd/blob/08528c28b7ef2d3ce59ea9a79579a01ac5d5cded/hdkeychain/extendedkey_test.go#L410-L417) and the BIP32 variant is tested via the new `TestBIP0032Vector4`.

Reply:
Killing this test.  It was mainly to convince myself that I got the same results of the test that btcutil added when they added the second derivation method.  https://github.com/btcsuite/btcutil/pull/182/files#diff-ecb1e43c8bedd7d0ff8580f1bf2c2685a1b34be9f50ab7710ad4954611cb1d4dR1161